modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -6118,17 +6125,47 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                
+                <xsd:attribute name="expired-and-dispatched-groups-object-store-ref" type="xsd:string" use="optional">
```

Comment:
Not sure if thee is other way to do this, but having all these object stores in the XML config does not seem user friendly. Could we use a partitionable object store and create partitions for each different task?